src/hooks/useSubmitTransaction.ts

```diff
@@ -39,7 +39,7 @@ export function useSubmitTransaction(
       });
     },
     onError(error) {
-      console.log('Transaction failed with args\n:', JSON.stringify(contractConfig.args));
+      console.log('createSarcophagus failed with args\n:', JSON.stringify(contractConfig.args));
```

Comment:
This isn't specifically a createSarcophagus transaction, although it was only being used for it, until a couple merges ago

Reply:
good call! fixing